radiopadre/__init__.py

```diff
@@ -144,28 +144,27 @@ def _init_js_side():
         print("get_ipython not found")
         return None
     get_ipython().magic("matplotlib inline")
-    # load radiopadre/js/init.js and init controls
-    #initjs = os.path.join(os.path.dirname(__file__), "html", "init-radiopadre-components.js")
-    #display(Javascript(open(initjs).read()))
 
     settings.display.reset = _display_reset, settings_manager.DocString("call this to reset sizes explicitly")
 
-    #warns = "\n".join([render_status_message(msg, bgcolor='yellow') for msg in radiopadre_kernel._startup_warnings])
-
     html = """<script type='text/javascript'>
             document.radiopadre.register_user('{}');
             document.radiopadre.reset_display_settings();
             </script>
          """.format(os.environ['USER'])
 
-    styles_file = os.path.join(os.path.dirname(__file__), "html/radiopadre.css")
 
-    html += """<style>
-        {}
-    </style>""".format(open(styles_file).read())
+    # reload styles -- loaded from radiopadre-kernel.js already, but reloading is useful for debugging
+    styles_file = os.path.join(os.path.dirname(__file__), "html/radiopadre.css")
```

Comment:
A small comment, could be: `os.path.join(os.path.dirname(__file__), "html", "radiopadre.css")`

Reply:
Could be. But the chances of this ever being ported to Windows are 1e-6%...